src/public/index.php

```diff
@@ -77,11 +77,9 @@ function handle_exception(\Throwable $e)
 
 $route = $router->getRoute($request);
 $return_data = $route->dispatch($request, $ji_db, $container);
-
-if ($return_data && isset($request->user_id)) {
+if (is_array($return_data) && isset($request->user_id)) {
```

Comment:
It's definitely better than before, but shouldn't we also check whether the array-key is set?

```php
if (isset($return['meta']['user_uri']) && isset($request->user_id)) {
```

Reply:
stupid me. Checking whether it's an array is perfeclty sufficcient sa the key is created if it doesn't exist.... :man_facepalming: